dev-packages/e2e-tests/maestro/feedback/captureFlow-android.yml

```diff
@@ -0,0 +1,50 @@
+# This is a happy path test for the feedback widget on Android.
+# It verifies that the feedback form can be opened, filled out, and submitted successfully
+appId: ${APP_ID}
+jsEngine: graaljs
+---
+
+# Show feedback button
+- tapOn: 'Feedback'
+
+# Open feedback widget
+- tapOn: 'Report a Bug'
+
+# Assert that the feedback form is visible
+- extendedWaitUntil:
+    visible: 'Report a Bug'
+    timeout: 5_000
+
+# Fill out name field
+- tapOn: 'Your Name'
+- inputText: 'John Doe'
+
+# Fill out email field
+- tapOn: 'your.email@example.org'
+- inputText: 'test@email.com'
+
+# Fill out message field
+- tapOn: "What's the bug? What did you expect?"
+- inputText: 'This is a test feedback message with a screenshot from CI e2e tests'
+
+# Take screenshot
+- scrollUntilVisible:
+    element:
+      text: 'Take a screenshot'
+- tapOn: 'Take a screenshot'
+- tapOn: 'Take Screenshot'
+
+# Hide keyboard
+- tapOn:
+    id: 'sentry-logo'
```

Comment:
Tapping on a non-tappable region is a work around to hide the keyboard since:
- [On Android sometimes the `hideKeyboard` command may trigger a back navigation](https://github.com/mobile-dev-inc/maestro/issues/1303)
- [On iOS the `hideKeyboard` command is flaky](https://github.com/mobile-dev-inc/maestro-docs/blob/main/troubleshooting/known-issues.md#hidekeyboard-command-is-flaky)